ts/routes/graphs/reviews.ts

```diff
@@ -109,19 +108,36 @@ export function renderReviews(
             break;
     }
     const desiredBars = Math.min(70, Math.abs(xMin!));
-
-    const x = scaleLinear().domain([xMin!, xMax]);
-    if (range === GraphRange.AllTime) {
-        x.nice(desiredBars);
+    const shouldCapRange = range !== GraphRange.AllTime;
+    const originalXMin = shouldCapRange ? xMin! : undefined;
+
+    // Create initial scale to determine tick spacing
+    let x = scaleLinear().domain([xMin!, xMax]);
+    // Generate thresholds: these define where bins are split (e.g., [-5, -10, -15, ...])
+    // D3's bin() will create bins between consecutive thresholds: [threshold[i], threshold[i+1])
+    let thresholds = x.ticks(desiredBars);
+    // Adjust xMin to align with tick spacing so the oldest bin has the same width as others
+    if (thresholds.length >= 2) {
+        const spacing = thresholds[1] - thresholds[0];
+        const partial = thresholds[0] - xMin!;
+        if (spacing > 0 && partial > 0 && partial < spacing) {
+            // Extend xMin backward to align with tick spacing, but cap at originalXMin for fixed ranges
+            xMin = Math.max(thresholds[0] - spacing, shouldCapRange ? originalXMin! : -Infinity);
+            x = scaleLinear().domain([xMin, xMax]);
+            thresholds = x.ticks(desiredBars);
+        }
+    }
+    // For Year, shift thresholds forward by one day to make first bin 0-4 instead of 0-5
+    if (range === GraphRange.Year) {
```

Comment:
```suggestion
    if (range === GraphRange.Year || range === GraphRange.AllTime) {
```

This fixes the following issue:
In "all", the latest bin says "0-20 days" ago (i.e., it includes 21 days) instead of the expected "0-19 days" (because all the other bins include 20 days).